File: RAI/Clase 14/Java/CasaCambio.java
/**
 * Ubicación  : 'POO-PA2022-1/RAI/Clase 14/'
 * Compilación: javac Java/CasaCambio.java
 * Ejecución  : java Java.CasaCambio 
 */

package Java;

public class CasaCambio {
    
    public static void main(String[] args){
        Moneda dolar = new Moneda("usd", "USA", 1);
        Moneda euro = new Moneda("eur", "EU", 0.93);
        Moneda libra = new Moneda("libra", "UK", 0.78);
        Moneda peso = new Moneda("clp", "Chile", 845);
    
        double monto = 1;
        System.out.println("> " + monto + " "+ dolar.getNombre() + " son " + peso.cambiarPor(monto,dolar) + " " + peso.getNombre());
    }
    
}

File: RAI/Clase 14/Python/CasaCambio.py
from Divisa import Divisa
from Moneda import Moneda

if __name__ == "__main__":
    dolar = Moneda("usd", "USA", 1)
    euro = Moneda("eur", "EU", 0.93)
    libra = Moneda("libra", "UK", 0.78)
    peso = Moneda("clp", "Chile", 845)
    
    monto = 1
    print(f"> {monto} {dolar.get_nombre()} son {peso.cambiar_por(monto,dolar)} {peso.get_nombre()}:")